index.bs

```diff
@@ -621,6 +621,99 @@ The {{ML/createContextSync()}} method steps are:
 1. If the <a>validate MLContext</a> steps given |context| return `false`, throw a "{{NotSupportedError}}" {{DOMException}} and abort these steps.
 1. Return |context|.
 
+## The MLGraph interface ## {#api-mlgraph}
+The {{MLGraph}} interface represents a compiled computational graph that is defined in an {{MLContext}}. A compiled {{MLGraph}} once constructed is immutable and cannot be subsequently changed.
```

Comment:
No problem, I can remove that text for the context of this PR, but that text very much belongs there. It can be improved in another PR. No need to track it for now.

Reply:
Fixed, no other changes, only moving the text.